torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -603,7 +612,7 @@ class CppMicroGemmAMX(CppMicroGemm):
 
 {%- if input_dtype == torch.bfloat16 and input2_dtype == torch.int8 %}
     // create a buffer for tiles of B.
-    alignas(64) {{input_t}} bf16_weights_buf[512];
+    alignas(4096) {{input_t}} bf16_weights_buf[(K / 16) * 512];
```

Comment:
@jgong5 @leslie-fang-intel, please note that while this buffer is actually inside the inner-kernel, this code still works because the stack space allocated to the buffer is not modified between various invocations of the inner-kernel.
It's unlikely for any code changes in the GEMM micro-kernel template to change that behavior, since the micro-kernel template invokes it in succession.

However, if you'd like, I could move the buffer outside the inner-kernel. Thanks!

Reply:
Done